assignments/prototypes.js

```diff
@@ -32,79 +53,83 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(attr) {
+  CharacterStats.call(this, attr);
+  this.team = attr.team;
+  this.weapons = attr.weapons;
+  this.language = attr.language;
+}
+Humanoid.prototype = Object.create(GameObject.prototype);
```

Comment:
You shouldn't have to pull in the GameObject prototype. Once you pull in the CharacterStats prototype, the GameObject prototype should also be available because CharacterStats inherits GameObject.